packages/eslint-plugin/lib/config/yaml.ts

```diff
@@ -1,18 +1,11 @@
 import type { Linter } from "eslint";
-import { yamlFiles } from "../utils/patterns";
+import ymlPlugin from "eslint-plugin-yml";
+import { yamlFiles } from "../utils/patterns.ts";
 
-const config: Linter.Config = {
-    overrides: [
-        {
-            files: yamlFiles,
-            plugins: ["yml"],
-            extends: ["plugin:yml/recommended"],
-            parser: "yaml-eslint-parser"
-        }
-    ]
-};
+const config: Linter.FlatConfig[] =
+    ymlPlugin.configs["flat/recommended"].map(conf => ({
+        ...conf,
+        files: yamlFiles
+    }));
```

Comment:
Why are you using this "map" syntax instead of using a syntax similar to the other files?

Reply:
Because the YAML config already has a flat config, and I just need to map over it and modify the files. It doesn't need any other updates.